docs/docs/jaseci_architype/referencing_dereferencing.md

```diff
@@ -35,4 +35,10 @@ info: {
     "j_timestamp": "2022-08-10T15:57:00.577287", 
     "j_type": "node"
 }
-```
\ No newline at end of file
+```
+
+The code snippet creates a simple node with an attribute named name. It then initializes a walker named ref_deref, which creates three new nodes named node0, node1, and node2 and assigns them to the simple node.
+
+The code then assigns the reference of the first node node0 to the var variable using the & operator. The std.out() function is used to print out the reference, object, and information related to the var variable.
```

Comment:
use backtick keys to highlight lines of codes inside paragraphs. like this `std.out()`

Reply:
Thanks for this recommendation, I actually went through the docs and noticed that there are multiple occurrences of this.